### PR TITLE
Sensor Overflow Logic fix

### PR DESCRIPTION
**Pull request by Rvice**
Issue #5 fix by adding the maxCount and using a >= instead of just == check.

See Control Register (0x01) MaxCount for ATime of 100 ms to see the max count from the sensor is 36863

### CODE REVIEW

File: adafruit_tsl2591.py [hunk 1]
```diff
@@ -69,6 +69,8 @@
 _TSL2591_LUX_COEFB           = 1.64
 _TSL2591_LUX_COEFC           = 0.59
 _TSL2591_LUX_COEFD           = 0.86
+_TSL2591_MAX_COUNT_100MS     = const(36863)
```
Comment (tannewt) [inline adafruit_tsl2591.py]: Please rewrite this number as 0x8fff to match the one below.

**Review by tannewt — CHANGES_REQUESTED** [review]
One small thing then it'll be good to go. Thanks!

File: adafruit_tsl2591.py (hunk 1 above)
Comment (Rvice) [reply adafruit_tsl2591.py]: Any aversion to writing the one below to 65535 to reflect what is in the datasheet. This way one can search for 36863 and find it in the datasheet as well as 65535, whereas the hex values are nowhere to be found in the datasheet.

See Control Register (0x01) in the TSL2591 datasheet

Comment (dhalbert) [reply adafruit_tsl2591.py]: How about writing one as a comment and one as a constant?

Comment (Rvice) [reply adafruit_tsl2591.py]: Put the comments in for the hex representation as well

**Review by tannewt — APPROVED** [review]
Awesome! Thank you so much!

**Pull request merged**